authors/urls.py

```diff
@@ -22,6 +22,7 @@
 urlpatterns = [
     path('admin/', admin.site.urls),
     path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication')),
-    path(r'', schema_view)
+    path(r'', schema_view),
+    path('api/', include(('authors.apps.profiles.urls', 'profiles'), namespace='profiles')),
```

Comment:
This is good, but It would be better if the line `path('api/', include(('authors.apps.profiles.urls', 'profiles'), namespace='profiles')),` came after `path('api/', include(('authors.apps.authentication.urls', 'authentication'), namespace='authentication'))`, grouping the api endpoints together looks better and easy to find what you looking for.

I'd say put `path(r'', schema_view),` before the the admin endpoint, and the api endpoints after admin endpoint